magic-8-ball/apps/01-getting-started/hello-python/spin.toml

```diff
@@ -0,0 +1,17 @@
+spin_manifest_version = 2
+
+[application]
+authors = ["Sohan <1119120+sohanmaheshwar@users.noreply.github.com>"]
```

Comment:
Small nit: I've been using the engineering email for the authors in all the sample apps.

Reply:
yes let's use the engineering one